source/_docs/distros.md

```diff
@@ -0,0 +1,94 @@
+---
+title: New Sites and Public Distributions
+description: Learn how to create sites using a public distribution start state.
+tags: [create]
+---
+The Pantheon Dashboard provides a quick "click to install" method of creating new sites that utilize our default, "vanilla" frameworks (e.g., WordPress, Drupal 7, and Drupal 8). However, it's possible to create sites that use an available public distribution (e.g., Panopoly) with Terminus, the Pantheon CLI.
```

Comment:
+1 I didn't realize that direct links are available. Do we have a list of them, or a format to follow so I can construct them?